esp8266-fastled-webserver/esp8266-fastled-webserver.ino

```diff
@@ -1389,7 +1399,7 @@ void fireFibonacci() {
     uint16_t x = coordsX[i];
     uint16_t y = coordsY[i];
 
-    uint8_t n = qsub8( inoise8((x << 2) - beat88(speed << 2), (y << 2)), x );
+    uint8_t n = qsub8(inoise8((y << 2) - beat88(speed << 2), (x << 2)), y);
```

Comment:
Was this something you found more pleasing to the eye?
Were different branches swapping the use of X/Y? 
Was this a regression I added?

I just might not have noticed the difference between branches.   I'm 100% approving this x/y swap, just wondering...

Reply:
Yeah, it fixed the orientation. Previously, the fire rose and the water fell sideways. I've tested this change on F256 and F64, but it may end up differing on other models and need to be configurable.